src/backend/partaj/core/api/unit_lite.py

```diff
@@ -0,0 +1,57 @@
+"""
+Topic lite related API endpoints.
+"""
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from ..forms import UnitListQueryForm
+from ..indexers import ES_CLIENT, UnitsIndexer
+from .common import AutocompleteMixin, ViewSetMetadata
+
+
+class UnitLiteViewSet(AutocompleteMixin, viewsets.GenericViewSet):
+    """
+    API endpoints for lite units.
+    Uses ElasticSearch to always return lists of lite units, accessible to all
+    authenticated users.
+    """
+
+    _meta = ViewSetMetadata(indexer=UnitsIndexer)
+
+    permission_classes = [IsAuthenticated]
+
+    def list(self, request, *args, **kwargs):
+        """
+        Handle requests for lists of lite units (except autocomplete).
+        """
+        form = UnitListQueryForm(data=self.request.query_params)
+        if not form.is_valid():
+            return Response(status=400, data={"errors": form.errors})
+
+        es_query_filters = []
+
+        _id = form.cleaned_data.get("id")
```

Comment:
What kind of id is it ? This point is a bit blurry for me.

Reply:
This is a list of IDs to filter the request for units on. The client sends `?id=1&id=4&id=65`, then `_id == ["1", "4", "65"]`.

The form is just used to validate the params and pre-format the data for us.